Lombiq.NodeJs.Extensions/scripts/markdown-to-msbuild.js

```diff
@@ -0,0 +1,129 @@
+﻿const findRecursively = require('./find-recursively');
+const fs = require('fs');
+const fsPromises = require('node:fs/promises');
+const markdownlint = require('markdownlint');
+const path = require('path');
+const process = require('process');
+const textlint = require("textlint");
+
+const markdownlintConfig = {
+    default: true,
+    MD013: false,
+    MD033: {
+        allowed_elements: [
+            // A special element in GitHub to indicate a keyboard key. Other Markdown formatters that don't support it will
+            // safely ignore the tags and render the content as inline text without adverse effects.
+            'kbd',
+        ]
+    }
+};
+const textLintConfig = {
+    exclude: [
+        // License files are full of legalese, which can't and shouldn't be analysed with tools made for normal prose.
+        'License.md',
+        // The wwwwroot directory contains built and vendor assets. Any Markdown file there is not our responsibility.
+        'wwwroot'
+    ],
+    rules: [
+        "common-misspellings",
+        "doubled-spaces",
+        //"no-dead-link", // Disabled becuase it can't ignore relative links and can't reliably verify GitHub URLs.
+        "no-todo",
+        "no-zero-width-spaces",
```

Comment:
Are these only rule overrides, or all the rules that are applied? Because there are a lot more rules in textlint that we should probably utilize.

Reply:
There is no English preset so I just added whatever felt right. I considered adding [rousseau](https://github.com/azu/textlint-rule-rousseau) too, but then I'd have to rewrite pretty much every other sentence to avoid "passive voice" and other weird stuff that felt like it's trying to force literary style on a technical document. When I took on this issue I didn't expect that it requires a proofreader (lektor) and I'm not one.
If you have specific rules you want to enable, let's discuss.

Reply:
I'm talking about all the other base and English rules here: https://github.com/textlint/textlint/wiki/Collection-of-textlint-rule

Reply:
Certainly not a _all_ of them. A bunch of them overlap (e.g.  textlint-rule-rousseau and textlint-rule-write-good essentially do the same) and lot of them also require additional resources (python, Java, hunspell, external paid API for Ginger, etc). Also I think [we have already agreed](https://teams.microsoft.com/l/message/19:98bc0e0e8a674128a7710e618d04d0e6@thread.skype/1658141940206?tenantId=959ede5a-38e9-44e6-adc8-85553beb8220&groupId=67839e8e-8ec5-44da-8188-a14c4c6af065&parentMessageId=1658137059516&teamName=Lombiq%20Team&channelName=Lombiq%20limbo&createdTime=1658141940206) not to use at least one of them.

I'm also against using any traditional spellchecker rule that relies on dictionaries (e.g. textlint-rule-spellchecker, textlint-rule-en-spell) because they never include all the domain and project specific words so we'd have to maintain separate dictionaries for every closed source project.

Reply:
How about the ones not requiring anything else external but no referenced here, e.g. `textlint-rule-no-start-duplicated-conjunction`, `textlint-rule-max-comma`, `textlint-rule-no-empty-section`, or `textlint-rule-editorconfig`? To put it otherwise, apart from spellcheckers, why are we not utilizing anything else apart from these four rules?

Reply:
why use `textlint-rule-editorconfig` for markdown files?

Reply:
I've added the rest and applied the fixes.

Reply:
Great!